test.sh

```diff
@@ -63,7 +63,8 @@ else
   run go test ${dirlist}
 fi
 
-echo "Checking for unformatted files:"
+setsid bash test/integration-test.sh
```

Comment:
I think the only thing that still needs doing is to check for the existence of setsid before attempting to use it, since it doesn't exist on BSD. I'd recommend doing something like:

```
SETSID=$(which setsid)
${SETSID} bash test/integratio-test.sh
```